scripts/docker/debian/jessie/Dockerfile

```diff
@@ -51,6 +51,15 @@ RUN apt-get update && apt-get -y install \
 
 RUN cabal update && cabal install hspec QuickCheck
 
+# Google Test
```

Comment:
Thank you for adding this 💖!

Reply:
Why not installing the Debian package "googletest"?

From the description of the package:
"NOTE: This package does not contain a library to link against, but rather the source code to build the google test library; this is because the google test library must be built with the same flags as the C++ code under test."

Is it okay if I modify scripts/debian* to include `-DGTEST_ROOT="/usr/src/googletest/googletest" \`

Reply:
because that will most definitely not be the version that is autodownloaded.

Reply:
Yes, but it is the version used when building Debian packages. We can use different googletest in different jobs. We should state somewhere which versions will work anyway.

Reply:
> We should state somewhere which versions will work anyway.

More or less every version beginning with Google Test `1.7` should work. I used a recent version, since Google Test `1.7` is five years old and Google Test `1.8` will produce CMake warnings.

Reply:
https://github.com/ElektraInitiative/libelektra/pull/2003/commits/f225050523f4835bd59eb232194753b7272bddab